### ISSUE TITLE
gitnuro: missing libGL at runtime

### ISSUE THREAD

**Issue by sheepforce** [NixOS/nixpkgs#302550]
### Describe the bug
Gitnuro crashes immediately with a missing libGL error at runtime.

### Steps To Reproduce
1. `nix run nixpkgs#gitnuro`

```
After reading prefs, got 3 tabs
TabViewModel - Trying to open repository /home/phillip/Dokumente/Git/KeyToXml
TabViewModel - Trying to open repository /home/phillip/Dokumente/Git/compix
TabViewModel - Trying to open repository /home/phillip/Dokumente/Git/NixOps
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Exception in thread "main" java.lang.UnsatisfiedLinkError: /home/phillip/.skiko/f45c402aa4e9af92056695052a7d660cade0c952f4f82227886405caa27ebdf4/libskiko-linux-x64.so: libGL.so.1: cannot open shared object file: No such file or directory
```
(and a long java trace to follow)

### Expected behavior
LibGL should be found and gitnuro should not crash

### Additional context
Still works in NixOS 23.05

### Notify maintainers
@zendo

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.78, NixOS, 23.11 (Tapir), 23.11.20240311.ddcd759`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(phillip): `"nixpkgs"`
 - nixpkgs: `/home/phillip/.nix-defexpr/channels/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by zendo** [comment]
fixed: https://github.com/NixOS/nixpkgs/pull/278514

**Comment by sheepforce** [comment]
Closed by #278514
